javascript-data-model/order-history.js

```diff
@@ -0,0 +1,27 @@
+var orderHistory = [
```

Comment:
so to be clear, what I sent was an example of the structure you are aiming for, not the actual structure. So make sure to remake this in this style but using the order info contained in the example image.